docs/release-notes/.FSharp.Compiler.Service/9.0.100.md

```diff
@@ -27,4 +27,7 @@
 * Enforce `AttributeTargets` on unions. ([PR #17389](https://github.com/dotnet/fsharp/pull/17389))
 * Ensure that isinteractive multi-emit backing fields are not public. ([Issue #17439](https://github.com/dotnet/fsharp/issues/17438)), ([PR #17439](https://github.com/dotnet/fsharp/pull/17439))
 * Enable FSharp 9.0 Language Version ([Issue #17497](https://github.com/dotnet/fsharp/issues/17438)), [PR](https://github.com/dotnet/fsharp/pull/17500)))
+* Enable graph type checking and parallel optimizations by default, can be turned off via the following compiler flags `--graphtypechecking-` and `--paralleloptimization-` respectfully. ([PR #17442](https://github.com/dotnet/fsharp/pull/17442))
```

Comment:
```suggestion
* Enable graph type checking and parallel optimizations by default, can be turned off via the following compiler flags `--graphtypechecking-` and `--paralleloptimization-` respectively. ([PR #17442](https://github.com/dotnet/fsharp/pull/17442))
```

:)

Reply:
Maybe I want people to turn it on with respect? :)